### ISSUE TITLE
we agree stm32 doesn't work with tmc2130 in spi

### ISSUE THREAD

**Issue by maximeh2** [grblHAL/STM32F4xx#174]
it's currently in experimental mode?

**Comment by terjeio** [comment]
Yes kind of still is, but should be working. Any feedback will be appreciated - I simply do not have time, or hardware, to test every configuration extensively.
